src/main/java/org/broadinstitute/hellbender/utils/read/BDGAlignmentRecordToGATKReadAdapter.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.bdgenomics.adam.converters.AlignmentRecordConverter;
+import org.bdgenomics.adam.models.SAMFileHeaderWritable;
+import org.bdgenomics.formats.avro.AlignmentRecord;
+
+import java.util.UUID;
+
+/**
+ * Implementation of the {@link GATKRead} interface for the {@link AlignmentRecord} class.
+ *
+ * This adapter wraps an {@link AlignmentRecord} by first converting to a {@link SAMRecord} and then passing through
```

Comment:
Does this exist somewhere for the Google `Read` object?  I could put it in an analogous place.  Otherwise, I'll just put it here.